decidim-assemblies/app/models/decidim/assembly.rb

```diff
@@ -54,5 +57,17 @@ def hashtag
     def to_param
       slug
     end
+
+    def ancestors
```

Comment:
Should we save these ancestors as an array of IDs in the DB, so we can speed up the relation? If we save an array, then we can do `Assembly.where(id: ancestor_ids)` and we can return them all at once instead of querying them one by one